samples/xmlserializergenerator-instructions.md

```diff
@@ -45,6 +36,6 @@ Here are the step by step instructions on how to use Xml Serializer Generator in
     ```c#
     var serializer = new System.Xml.Serialization.XmlSerializer(typeof(MyClass));
     ```    
-5. Build the application by running `dotnet build`. If everything succeeds, an assembly named MyApp.XmlSerializers.dll will be generated    in the output folder. You will see warnings in the build output if the tool failed to generate the assembly.
+4. Build the application by running `dotnet build`. If everything succeeds, an assembly named MyApp.XmlSerializers.dll will be generated    in the output folder. You will see warnings in the build output if the tool failed to generate the assembly.
```

Comment:
Remove the space between generated  "will be generated    in the output folder"

Reply:
Done.